Tasks/DeployAzureResourceGroup/task.json

```diff
@@ -116,6 +116,24 @@
             "visibleRule": "action = Create Or Update Resource Group"
         },
         {
+            "name": "enableRemoteDeploymentForCreate",
+            "type": "boolean",
+            "label": "Enable Remote Deployment",
+            "defaultValue": "true",
+            "visibleRule": "action = Create Or Update Resource Group",
```

Comment:
Does VisibleRule have an Or notion where you can specify both create and select actions?